effekt/shared/src/main/scala/effekt/symbols/symbols.scala

```diff
@@ -296,6 +298,7 @@ enum BlockTypeConstructor extends BlockTypeSymbol {
 
   case Interface(name: Name, tparams: List[TypeParam], var operations: List[Operation] = Nil)
   case ExternInterface(name: Name, tparams: List[TypeParam])
+  case ErrorBlockType(name: Name = NoName, tparams: List[TypeParam] = Nil)
```

Comment:
<img width="383" alt="Screenshot 2025-04-22 at 21 55 49" src="https://github.com/user-attachments/assets/0195adfc-17d4-4a7b-8a78-31b93ab5715a" />
Using `NoName` here shows `<anon>` when hovering and I can't see where to fix that except for making a copy of `NoName` called `ErrorName` or something...

Reply:
Moving this into `BlockType` might help since then you can define a custom pretty printer for it.